src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -2198,7 +2199,21 @@ def serialize_key_and_certificates_to_pkcs12(
             # This list is to keep the x509 values alive until end of function
             ossl_cas = []
             for ca in cas:
-                ossl_ca = self._cert2ossl(ca)
+                if isinstance(ca, PKCS12Certificate):
+                    # Copied from Felix's gist
+                    # https://gist.github.com/
+                    # felixfontein/f750763fd5773fbf0ab22af87239aab0
```

Comment:
No need for this citation :-)

Reply:
Left it just to make sure, removed it now :)